test/data_handling/parsers/test_forty_two_parser.py

```diff
@@ -68,7 +68,7 @@ def test_FortyTwo_parse_sim_data_raises_index_error_when_given_empty_data_file(m
 
```

Comment:
I think you missed one modker.patch at line 34:

    mocker.patch("onair.data_handling.parsers.forty_two_parser.str2lst", return_value=forced_return_str2lst)

Reply:
yep, fixed in latest push